### PR TITLE
Update minimum required version of Xcode and Swift. Update styling example.

### PR DESCRIPTION
**Pull request by MaximAlien**
### Description

Update minimum required version of Xcode and Swift. Update styling example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,8 @@ class CustomStyle: DayStyle {
 
     override func apply() {
         super.apply()
-        BottomBannerView.appearance().backgroundColor = .orange
```
Comment (MaximAlien) [inline README.md:181]: It'd be better to move styling changes to separate PR, but since they're so small and more or less related (mostly docs), I merged them into one PR.

**Review by chezzdev — APPROVED** [review]
